models/users.py

```diff
@@ -21,9 +21,9 @@
 class User(db.Model, UserMixin):
     __tablename__ = 'users'
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(128), unique=True)
-    first_name = db.Column(db.String(64))
-    surname = db.Column(db.String(64))
+    email = db.Column(db.String(128), unique=True, nullable=False)
```

Comment:
not sure about that.
Why nullable was proposed ?

Reply:
My form uses validation from the db model. I added nullable=False in order to prevent user deleting his data during profile edition.